cli/core/status.go

```diff
@@ -136,56 +154,68 @@ func GetStatus(ctx context.Context, eigenpodAddress string, eth *ethclient.Clien
 	PanicOnError("failed to get eigenpod proof submitter", err)
 
 	currentOwnerShares, err := eigenPodManager.PodOwnerShares(nil, eigenPodOwner)
+	// currentOwnerShares = big.NewInt(0)
 	PanicOnError("failed to load pod owner shares", err)
 	currentOwnerSharesETH := IweiToEther(currentOwnerShares)
+	currentOwnerSharesGwei := WeiToGwei(currentOwnerShares)
 
 	withdrawableRestakedExecutionLayerGwei, err := eigenPod.WithdrawableRestakedExecutionLayerGwei(nil)
 	PanicOnError("failed to fetch withdrawableRestakedExecutionLayerGwei", err)
 
-	var pendingSharesGwei *big.Float
-	mustForceCheckpoint := false
-	// If we currently have an active checkpoint, estimate the total shares
-	// we'll have when we complete it:
+	// Estimate the total shares we'll have if we complete an existing checkpoint
+	// (or start a new one and complete that).
 	//
-	// pendingSharesGwei = withdrawableRestakedExecutionLayerGwei + checkpoint.PodBalanceGwei + sumRegularBalancesGwei
-	if timestamp != 0 {
-		pendingSharesGwei = new(big.Float).Add(
-			new(big.Float).Add(
-				new(big.Float).SetUint64(withdrawableRestakedExecutionLayerGwei),
-				new(big.Float).SetUint64(checkpoint.PodBalanceGwei),
-			),
-			new(big.Float).SetUint64(uint64(sumRegularBalancesGwei)),
-		)
+	// First, we need the change in the pod's native ETH balance since the last checkpoint:
+	var nativeETHDeltaGwei *big.Float
+	mustForceCheckpoint := false
+
+	if checkpointTimestamp != 0 {
+		// Change in the pod's native ETH balance (already calculated for us when the checkpoint was started)
+		nativeETHDeltaGwei = new(big.Float).SetUint64(checkpoint.PodBalanceGwei)
```

Comment:
PodBalanceGwei is the delta?

Reply:
yep! it's calculated when the checkpoint is started on chain

Reply:
probably couldve been named better :grimacing: